internal/console/generate_id_test.go

```diff
@@ -0,0 +1,56 @@
+package console
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+type execute func() error
+
+func captureOutput(funcToExec execute) (string, error) {
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	err := funcToExec()
+
+	outC := make(chan string)
+
+	go func() {
+		var buf bytes.Buffer
+		io.Copy(&buf, r) // nolint
+		outC <- buf.String()
+	}()
+
+	w.Close()
+	os.Stdout = old
+	out := <-outC
+
+	return out, err
+}
+
+func captureCommandOutput(cmd *cobra.Command, args []string) (string, error) {
+	if len(args) > 0 {
+		cmd.SetArgs(args)
+	}
+
+	return captureOutput(cmd.Execute)
+}
```

Comment:
Move this code to a helper file, maybe inside /tests

Reply:
Moved to test/helpers.go